tsconfig.json

```diff
@@ -25,5 +25,5 @@
     "strict": true,
     "target": "esnext"
   },
-  "exclude": ["example"]
+  "exclude": ["example"] // why??
```

Comment:
https://github.com/callstack/react-native-pager-view/blob/master/tsconfig.build.json

Reply:
Sorry about this one, it was meant as a reminder for me to investigate this `exclude` further, but it slipped my mind and ended up in the PR 😁 I removed it, but to answer your comment — thanks for clarifying, I understand where the value is coming from, but I was wondering about why we're not type checking the example folder, time savings? (If you remove this `exclude` there are some 200+ new TS errors emitted.)